src/routes/room.route.js

```diff
@@ -1,7 +1,8 @@
 const router = require('express').Router()
 const roomController = require('../controllers/room.controller')
 
-router.post('/create', roomController.create);
+
+router.post('/create', roomController.createRoom);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Backend/pullRequest?prid=7196401)
```suggestion
router.post("/create", roomController.createRoom);
```